src/routes/+layout.svelte

```diff
@@ -45,5 +45,9 @@
 		inset: 0 0 auto 0;
 		height: 28px;
 		z-index: 9999;
+		cursor: grab;
+		&:active {
+			cursor: grabbing;
+		}
```

Comment:
We probably don't want this because it is not the case in other apps and the user won't expect it